### PR TITLE
Update CHANGELOG and README for v1.1.0

### LINKED ISSUE #334: Release bem-core@1.1.0
\cc @dfilatov @veged 

It seems that there are enough changes was made [since 1.0 release](https://github.com/bem/bem-core/compare/v1.0.0...v1). Guess we could move to next minor version.

Plan:
- [x] Update CHANGELOG
- [ ] <del>Write the migration guide</del>

### PR DESCRIPTION
**Pull request by narqo**
closes #334 

\cc @veged @dfilatov @tadatuta check, plz

### CODE REVIEW

**Comment by veged** [comment]
maybe we can reorder major items by "criticalness"/"importance" (from top to bottom), for example #292 < #326 — but this is minor fixes and we can skip it in case of controversial opinions

in general, looks ok to me

**Comment by dfilatov** [comment]
There is nothing about splitting of default version jquery for old and modern browsers.

**Comment by narqo** [comment]
> maybe we can reorder major items by "criticalness"/"importance" [..]

@veged I tried to sort this list, but from my point of view, fixing #292 is much critical then updating `ym`. And still I think that it should be in a changelog, since, it could affect us (theoretically ;)

**Comment by veged** [comment]
ok — no problems ;-)-- 
  Сергей Бережной

**Comment by tadatuta** [comment]
can we merge and include into `1.1.0` these PRs (or just some of them):
https://github.com/bem/bem-core/pull/341
https://github.com/bem/bem-core/pull/343
https://github.com/bem/bem-core/pull/330
?

File: CHANGELOG.md [hunk 1]
```diff
@@ -56,7 +80,7 @@
 - `trigger` method from `i-bem` block is **deprecated** in flavor of `emit` method.
 - `onFirst` method from `i-bem` block is **deprecated** in flavor of `once` method.
 - **Deprecated** field `e.block` that provided block-target of BEM-events was removed. Use `e.target` field instead.
-- Field `e.domElem` that provided DOM-element of block in DOM-events was removed. Use `$(e.currentTarget)` (provided by jQuery).
+- Field `e.data.domElem` that provided DOM-element of block in DOM-events was removed. Use `$(e.currentTarget)` (provided by jQuery).
```
Comment (narqo) [inline CHANGELOG.md:85]: @dfilatov am i right?

Comment (dfilatov) [inline CHANGELOG.md:85]: yes, definitely

**Comment by narqo** [comment]
Guess we could merge this after #343

**Comment by tadatuta** [comment]
merged https://github.com/bem/bem-core/pull/343

**Pull request merged**